quickcheck-dynamic/src/Test/QuickCheck/StateModel.hs

```diff
@@ -498,6 +549,14 @@ runActions
   => Actions state
   -> PropertyM m (Annotated state, Env)
 runActions (Actions_ rejected (Smart _ actions)) = do
+  -- TODO: consider bucketing one level lower here - 0-10, 10-20, ... 100-200, 200-300, ... 1000-2000, ...
```

Comment:
I am not sure I understand this comment: one level lower than w.r.t what?

Reply:
it's been fixed now 😅